packages/components/src/tree-select/stories/index.tsx

```diff
@@ -33,15 +47,9 @@ const TreeSelectWithState = ( props ) => {
 	);
 };
 
-export const _default = () => {
-	const label = text( 'Label', 'Label Text' );
-	const noOptionLabel = text( 'No Option Label', 'No parent page' );
-	const hideLabelFromVision = boolean( 'Hide Label From Vision', false );
-	const help = text(
-		'Help Text',
-		'Help text to explain the select control.'
-	);
-	const tree = object( 'Tree', [
+export const Default = TreeSelectWithState.bind( {} );
+Default.args = {
+	tree: [
```

Comment:
Could we keep these demo values in the `Default.args`? Just so we get a good visual of the most commonly used props.

```suggestion
Default.args = {
	label: 'Label Text',
	noOptionLabel: 'No parent page',
	help: 'Help text to explain the select control.',
	tree: [
```